scapy/route.py

```diff
@@ -21,8 +22,12 @@
 
 class Route:
     def __init__(self):
-        self.resync()
-        self.cache = {}
+        if conf.route is not None and isinstance(conf.route, list):
```

Comment:
Again, when conf.route can be not None and not instanceof list? Wouldn't simpler check suffice?

Reply:
At least it could be rewritten as "if isinstance(conf.route, list)" - that would behave the same.

Reply:
These comments apply to subsequent code as well.

Reply:
No. conf.route can be a Route instance, in which case this does not work. It will only be triggered during startup.
When creating another Route instance (e.g in regression.uts tests) conf.route isn't a list

Reply:
That's weird. I don't think the first part of the test is needed.

Reply:
Yes the first part isn't needed. That was how it was before